vespa/json_serialization.py

```diff
@@ -37,7 +38,7 @@ def to_dict(self) -> typing.Mapping:
     @property
     def to_envelope(self) -> typing.Mapping:
         return {
-            "version": 1,
+            "version": get_distribution("pyvespa").version,
```

Comment:
@oyving Do you agree with this? I think it makes more sense to tag the serialized object with the pyvespa version used to create it instead of hardcode a version number.

Reply:
The version number here should be the version number of the format and not the version number of pyvespa. This makes it easier to see what deserialisation code to invoke later should you wish to support multiple versions of the format.

Reply:
But what if the `to_dict` and `from_dict` methods for a given class change in pyvespa version 0.3.5 for example. How will I track which version was used by old serialized objects?

Reply:
I guess it depends on the level of compatibility and strictness you want to follow. If you always make sure later versions understand earlier versions, then just comparing the version number of pyvespa will be fine. You might have to track version ranges inside `from_dict` if the format changes in some significant way in the future.

Reply:
Yes. Making sure later versions understand earlier versions is nice. But at the very least it would be nice to track which pyvespa version was used to serialize the object so that people can go back to earlier versions if needed. That is why I included `get_distribution("pyvespa").version`. 

Maybe have `library_version` in addition to the (serialization format) `version` and add it on the `from_dict` signature?
E.g.
```
mapping_class.from_dict(mapping["data"], mapping["library_version"])
```

Reply:
I think both are acceptable solutions. If we're not currently using the version number in the format for anything it would be OK to change it to be the library version and then handle those in code later should we need to.